sdk/src/types.ts

```diff
@@ -901,7 +901,7 @@ export type MakerInfo = {
 	maker: PublicKey;
 	makerStats: PublicKey;
 	makerUserAccount: UserAccount;
-	order: Order;
+	order?: Order;
```

Comment:
this is just for fully backwards compatibility?

Reply:
need it for spot still